Makefile

```diff
@@ -20,9 +20,9 @@ REGISTRY_AND_USERNAME ?= $(REGISTRY)/$(USERNAME)
 PROTOBUF_GRPC_GATEWAY_TS_VERSION ?= 1.2.1
 TESTPKGS ?= ./...
 NODE_BUILD_ARGS ?=
-PROTOBUF_GO_VERSION ?= 1.34.1
+PROTOBUF_GO_VERSION ?= 1.33.0
 GRPC_GO_VERSION ?= 1.3.0
-GRPC_GATEWAY_VERSION ?= 2.20.0
+GRPC_GATEWAY_VERSION ?= 2.19.1
```

Comment:
These going down is expected I guess?

Reply:
Yeah. We might need to look into gRPC proxy to check if it supports that properly.
With these versions enabled all Omni API stops working.